src/lib/SeamComponents.tsx

```diff
@@ -100,3 +108,13 @@ const createSeamContextValue = ({
 const defaultSeamContextValue = createDefaultSeamContextValue()
 
 export const seamContext = createContext<SeamContext>(defaultSeamContextValue)
+
+export function useSeamComponents() {
```

Comment:
The original name was because it was a private hook. Why do we need this outside of this module (should it be part of the public API)?

Reply:
This hook just combines `useContext` with `seamContext`. `seamContext` was already previously exported (and `useContext`) was called separately.

I don't think there are any significant changes in terms of exposed API. Co-locating with the provider just makes it a little easier to use, and handles the case where it's called outside `<SeamProvider/>` by throwing an error.